### ISSUE TITLE
A11y_VisualStudioCodeClient_Editor_ Screen reader: Voiceover is announcing same information for both the suggestions for exclamation mark(!)

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#10738]
--- GitHub Tags:
-A11yMAS; -A11yTCS; -A11ySev2; -Device; -Visual Studio Code Client; -BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; -Voiceover; -Screenreader.

--- Environment and OS details:
Visual Studio Code
Screenreader: Voiceover

--- Repro Steps:

1. Open Visual studio code editor 
2. Create a HTML file.
3. Type (!) exclamation mark .
4. Suggestions will appear.
5. Turn on Voiceover
6. Verify that Voiceover is announcing same information for both the suggestions for exclamation mark(!).

--- Actual Result:
Voiceover is announcing same information for both the suggestions for exclamation mark(!).

--- Expected Result:
Voiceover should announce information related to the suggestion. Voiceover should not announce the same information for both the suggestions.

--- Attachment:
![exclamation](https://user-images.githubusercontent.com/98745784/198011683-5ba9bf72-3fd3-4932-a4a5-e74544fc3071.gif)
